### ISSUE TITLE
Material code is unreachable

### ISSUE THREAD

**Issue by raphaeltimbo** [ross-rotordynamics/ross#293]
In some `Material` methods we have the following code:
```python
def use_material(name):
    run_path = os.getcwd()
    ross_path = os.path.dirname(rs.__file__)
    os.chdir(ross_path)

    data = Material.load_data()
    try:
        material = data["Materials"][name]
        return Material(**material)
    except KeyError:
        raise KeyError("There isn't a instanced material with this name.")
    os.chdir(run_path)
```
The problem here is that the `os.chdir(run_path)` command will never run. In the try except statement the code will either suceed and return or it will raise a keyerror, never reaching the `os.chdir` command.

**Comment by raphaeltimbo** [comment]
One option would be to change the way we call the material methods.
Instead of changing the directory with `os.chdir`, we could give the complete path for the file:
```python
def available_materials():
    ross_path = Path(rs.__file__).parent / "available_materials.toml"

    try:
        data = Material.load_data(ross_path)
        return list(data["Materials"].keys())
    except FileNotFoundError:
        return "There are no saved materials."
```
The `load()` method would be something like:

```python
    def load_data(file_path):
        try:
            with open(file_path, "r") as f:
                data = toml.load(f)
        except FileNotFoundError:
            data = {"Materials": {}}
            Material.dump_data(data)
        return data
```
This way we would avoid changing directories back and forth.

**Comment by ross-bott** [comment]
Hi there!
I have marked this issue as stale because it has not had activity for 45 days.
Consider the following options:
- If the issue refers to a large task, break it in smaller issues that can be solved in
less than 45 days;
- Label the issue as `wontfix` or `wontfix for now` and close it.

**Comment by raphaeltimbo** [comment]
Solved by #432
